.github/workflows/create-release.yaml

```diff
@@ -23,6 +23,21 @@ on:
         type: boolean
         description: 'Do not publish'
         default: false
+  workflow_call:
+    inputs:
+      name:
+        type: string
+        description: 'Create release'
+        default: ""
+        required: true
+      sec-file-and-chart:
```

Comment:
I would rename it to `skip-sec-file-and-chart` or something like that, because now if sec-file-and-chart is false it may suggest that we don't want to create the bump.